runtime/src/iree/hal/command_buffer.h

```diff
@@ -359,6 +368,10 @@ static_assert(sizeof(iree_hal_collective_op_t) == sizeof(uint32_t),
 IREE_API_EXPORT iree_string_view_t iree_hal_collective_op_format(
     const iree_hal_collective_op_t* op, iree_bitfield_string_temp_t* out_temp);
 
+// Get the number of bytes each |element_type| consumes in memory.
```

Comment:
style nit: function comments start with verb phrases:
https://google.github.io/styleguide/cppguide.html#Function_Comments
```suggestion
// Returns the number of bytes each |element_type| consumes in memory.
```